client/src/actions/users.js

```diff
@@ -19,13 +19,15 @@ export const addUserAction = (userData) => async (dispatch) => {
     // We do not need to store the password, so create a new object without it and then dispatch
     const data = {
       email: userData.email,
-      givenName: userData.givenName,
-      familyName: userData.familyName
+      teamName: userData.teamName,
+      orgName: userData.orgName
     };
 
     // Store user related data in our redux global store
     dispatch({ type: ADD_USER_DATA, payload: data });
+    apiResult(true);
```

Comment:
I think it is better to return a boolean rather than passing a function like this. Wdyt?